src/Config.cpp

```diff
@@ -72,14 +72,13 @@ void _activateConfig();
 // reads the EEPROM until NUL terminator and returns the number of characters incl. terminator, starting from given address
 bool readConfigLength()
 {
-    char     temp       = 0;
     uint16_t addreeprom = MEM_OFFSET_CONFIG;
     uint16_t length     = MFeeprom.get_length();
     configLength        = 0;
 
     while (MFeeprom.read_byte(addreeprom++) != 0x00) {
         configLength++;
-        if (addreeprom > length) // abort if EEPROM size will be exceeded
+        if (addreeprom > length)                                       // abort if EEPROM size will be exceeded
```

Comment:
comment alignment. probably due to auto formatting rule.

This comment is also not necessary because it's obvious what is happening.

if `length` was called `eepromLength` it would be even more clear.

In general I propose to put comments above the line instead of behind.

Reply:
Do you want me to remove the comment as part of this PR? I was only trying to remove the temp variable, I didn't bother touching anything else.

Reply:
we can remove it - thanks